### PR TITLE
People Activity Now Has Working UI

### PR DESCRIPTION
**Pull request by joshkergan**

### CODE REVIEW

File: app/src/main/java/io/github/joshkergan/giftr/db/MappingContract.java [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 /**
  * Created by Patrick on 16/10/2016.
  * Contract for the Mapping Table, which maps the Item table to the People table
  */
 
 public final class MappingContract{
-    // No external instantiation
-    private MappingContract() {}
+	// No external instantiation
+	private MappingContract() {
```
Comment (pperrier27) [inline app/src/main/java/io/github/joshkergan/giftr/db/MappingContract.java]: Fix the formatting please

File: app/src/main/java/io/github/joshkergan/giftr/items/ItemContract.java [hunk 2]
```diff
@@ -8,15 +8,16 @@
  */
 
 public final class ItemContract{
-    // No external instantiation
-    private ItemContract() {}
+	// No external instantiation
+	private ItemContract() {
```
Comment (pperrier27) [inline app/src/main/java/io/github/joshkergan/giftr/items/ItemContract.java]: Fix the formatting please

File: app/src/main/java/io/github/joshkergan/giftr/items/ItemContract.java [hunk 3]
```diff
@@ -8,15 +8,16 @@
...
-    /* Inner class to define the table components */
-    public static class ItemEntry implements BaseColumns{
-        public static final String TABLE_NAME = "Item";
-        // This contains the item's full name, TYPE: STRING
-        public static final String COLUMN_NAME_ITEM = "itemName";
-        // Contains a reference to the photo stored for this item, TYPE: BLOB
-        public static final String COLUMN_NAME_PHOTO = "photo";
-    }
+	/* Inner class to define the table components */
+	public static class ItemEntry implements BaseColumns{
```
Comment (pperrier27) [inline app/src/main/java/io/github/joshkergan/giftr/items/ItemContract.java]: Let's figure out if we're using tabs or spaces

File: app/src/main/java/io/github/joshkergan/giftr/people/PeopleContract.java [hunk 4]
```diff
@@ -9,7 +9,8 @@
 
 public final class PeopleContract{
 	// No external instantiation
-	private PeopleContract() {}
+	private PeopleContract() {
```
Comment (pperrier27) [inline app/src/main/java/io/github/joshkergan/giftr/people/PeopleContract.java]: Fix the formatting please

**Review by pperrier27 — COMMENTED** [review]
Looks good, just some formatting changes required.

File: app/src/main/java/io/github/joshkergan/giftr/items/ItemContract.java (hunk 2 above)
Comment (bsankara) [reply app/src/main/java/io/github/joshkergan/giftr/items/ItemContract.java]: This is honestly personal preference, I'm okay with letting this be...

File: app/src/main/java/io/github/joshkergan/giftr/db/MappingContract.java (hunk 1 above)
Comment (bsankara) [reply app/src/main/java/io/github/joshkergan/giftr/db/MappingContract.java]: This is honestly personal preference, I'm okay with letting this be...

File: app/src/main/java/io/github/joshkergan/giftr/items/ItemContract.java (hunk 2 above)
Comment (bsankara) [reply app/src/main/java/io/github/joshkergan/giftr/items/ItemContract.java]: unless you mean the tabbing...

File: app/src/main/java/io/github/joshkergan/giftr/db/MappingContract.java (hunk 1 above)
Comment (pperrier27) [reply app/src/main/java/io/github/joshkergan/giftr/db/MappingContract.java]: We explicitly wanted this fixed in a previous commit, which is why I'm enforcing consistency here.

**Pull request merged**